cmd/nodecmd/loadtestStop.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	awsAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/aws"
+	gcpAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/gcp"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+func newLoadTestStopCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop [clusterName]",
+		Short: "(ALPHA Warning) Stops load test for an existing devnet cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode. 
+
+The node loadtest stop command stops load testing for an existing devnet cluster and terminates the 
+separate cloud server created to host the load test.`,
+
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(2),
+		RunE:         stopLoadTest,
+	}
+	return cmd
+}
+
+func stopLoadTest(_ *cobra.Command, args []string) error {
+	loadTestName := args[0]
+	clusterName := args[1]
+	var err error
+	existingSeparateInstance, err = getExistingLoadTestInstance(clusterName, loadTestName)
+	if err != nil {
+		return err
+	}
+	if existingSeparateInstance == "" {
+		return fmt.Errorf("no existing load test instance found in cluster %s", clusterName)
+	}
+	nodeConfig, err := app.LoadClusterNodeConfig(existingSeparateInstance)
+	if err != nil {
+		return err
+	}
+	switch nodeConfig.CloudService {
+	case constants.AWSCloudService:
+		_, separateHostRegion, err := getNodeCloudConfig(existingSeparateInstance)
+		if err != nil {
+			return err
+		}
+		loadTestEc2SvcMap, err := getAWSMonitoringEC2Svc(awsProfile, separateHostRegion)
+		if err != nil {
+			return err
+		}
+		if err = destroyNode(existingSeparateInstance, clusterName, loadTestName, loadTestEc2SvcMap[separateHostRegion], nil); err != nil {
+			return err
+		}
+	case constants.GCPCloudService:
+		var gcpClient *gcpAPI.GcpCloud
+		gcpClient, _, _, _, _, err = getGCPConfig(true)
+		if err != nil {
+			return err
+		}
+		if err = destroyNode(existingSeparateInstance, clusterName, loadTestName, nil, gcpClient); err != nil {
+			return err
+		}
+	default:
+		return fmt.Errorf("cloud service %s is not supported", nodeConfig.CloudService)
+	}
+	return nil
+}
+
+func destroyNode(node, clusterName, loadTestName string, ec2Svc *awsAPI.AwsCloud, gcpClient *gcpAPI.GcpCloud) error {
```

Comment:
we should split this func into separate func for AWS and GCP as we might have more clouds to support